state-chain/custom-rpc/src/lib.rs

```diff
@@ -371,10 +379,48 @@ mod boost_pool_details {
 			}
 		}
 	}
+
+	#[derive(Serialize, Deserialize)]
+	struct PendingFees {
+		deposit_id: PrewitnessedDepositId,
+		fees: Vec<AccountAndAmount>,
+	}
+
+	#[derive(Serialize, Deserialize)]
+	pub struct BoostPoolFeesRpc {
+		fee_tier: u16,
+		#[serde(flatten)]
+		asset: Asset,
+		pending_fees: Vec<PendingFees>,
+	}
+
+	impl BoostPoolFeesRpc {
+		pub fn new(asset: Asset, fee_tier: u16, details: BoostPoolDetails) -> Self {
```

Comment:
Jamie has a alias in his PR for this BoostPoolTier which is u16 @j4m1ef0rd .  Whichever PR gets in 2nd should update this.